src/main/java/com/sericulture/authentication/config/SecurityConfig.java

```diff
@@ -49,7 +49,8 @@ public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Excepti
                 .authorizeHttpRequests(auth ->
                         auth
                                 .requestMatchers(HttpMethod.OPTIONS,"/**").permitAll()
-                                .requestMatchers("/v3/api-docs/**","/auth/register","/auth/login","/auth/refresh-token","/swagger-ui/**","/login","/actuator/**","/v1/userMaster/generate-otp-by-user-name", "/v1/userMaster/verify-otp-by-user-name").permitAll()
+                                .requestMatchers("/v3/api-docs/**","/auth/register","/auth/login","/auth/refresh-token","/swagger-ui/**","/login","/actuator/health","/v1/userMaster/generate-otp-by-user-name", "/v1/userMaster/verify-otp-by-user-name").permitAll()
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 100 characters (found 262).